cmd/kor/hpas.go

```diff
@@ -0,0 +1,26 @@
+package kor
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/yonahd/kor/pkg/kor"
+)
+
+var hpaCmd = &cobra.Command{
+	Use:   "hpa",
+	Short: "Gets unused hpas",
+	Args:  cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		if outputFormat == "json" {
+			kor.GetUnusedHpasJson(namespace, kubeconfig)
+		} else {
+			kor.GetUnusedHpas(namespace, kubeconfig)
+		}
+	},
+}
+
+func init() {
+	hpaCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "k", "", "Path to kubeconfig file (optional)")
+	hpaCmd.PersistentFlags().StringVarP(&namespace, "namespace", "n", "", "Namespace to run on")
+	hpaCmd.PersistentFlags().StringVar(&outputFormat, "output", "table", "Output format (table or json)")
+	rootCmd.AddCommand(hpaCmd)
+}
```

Comment:
these flags seem to be getting set on every command, maybe they should be set at the root as `PersistentFlags` so they are available on every child command.

Reply:
honestly i followed the code base - didn't wanna draft too much from the other resources.
WDYT @yonahd  ?

Reply:
You can leave it this way now. I'll do the refactor across the code base.